### PR TITLE
backend sidebar contact model

### PR DESCRIPTION
**Pull request by kave**
https://github.com/cfpb/cfgov-refresh/issues/1363
screen shot 2016-02-10 at 10 31 44 am

@KimberlyMunoz 
@kurtw 
@richaagarwal

### CODE REVIEW

File: cfgov/v1/models/organisms.py [hunk 1]
```diff
@@ -106,6 +106,16 @@ class Meta:
         template = '_includes/organisms/main-contact-info.html'
 
 
+class SidebarContactInfo(blocks.StructBlock):
+    header = blocks.CharBlock(required=False)
```
Comment (anselmbradford) [inline cfgov/v1/models/organisms.py]: `content.header` is not wrapped in an `if` statement in `sidebar-contact-info.html`, should it be if this is not required?

Comment (kave) [inline cfgov/v1/models/organisms.py]: @anselmbradford good catch yea i'll add that in

File: cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
    See [GHE]/flapjack/Modules-V1/wiki/Contact-Information-(Sidebar)
 
    content:                  An object containing sidebar content.
-   content.heading:          A string containing a sidebar heading.
+   content.header:           A string containing a sidebar heading.
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html]: Update 'heading` in the description to `'header` to match

File: cfgov/v1/migrations/0045_auto_20160210_1530.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
....CharBlock(help_text=b'Type of form i.e emailForm, submission-form. Should be unique if multiple forms are used', required=False)), (b'info', wagtail.wagtailcore.blocks.RichTextBlock(required=False, label=b'Disclaimer')), (b'label', wagtail.wagtailcore.blocks.CharBlock(required=True)), (b'type', wagtail.wagtailcore.blocks.ChoiceBlock(required=False, choices=[(b'text', b'Text'), (b'checkbox', b'Checkbox'), (b'email', b'Email'), (b'number', b'Number'), (b'url', b'URL'), (b'radio', b'Radio')])), (b'placeholder', wagtail.wagtailcore.blocks.CharBlock(required=False))]), required=False, icon=b'mail'))])), (b'contact', wagtail.wagtailcore.blocks.StructBlock([(b'header', wagtail.wagtailcore.blocks.CharBlock(required=False)), (b'body', wagtail.wagtailcore.blocks.RichTextBlock(required=False)), (b'contact', wagtail.wagtailsnippets.blocks.SnippetChooserBlock(v1.models.snippets.Contact))])), (b'sidebar_contact', wagtail.wagtailcore.blocks.StructBlock([(b'header', wagtail.wagtailcore.blocks.CharBlock(required=False)), (b'body', wagtail.wagtailcore.blocks.RichTextBlock(required=False)), (b'contact', wagtail.wagtailsnippets.blocks.SnippetChooserBlock(v1.models.snippets.Contact))]))], blank=True),
```
Comment (jimmynotjim) [inline cfgov/v1/migrations/0045_auto_20160210_1530.py]: Can this line be broken on `,` to make it easier to read?

File: cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html (hunk 2 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html]: donzo

File: cfgov/v1/migrations/0045_auto_20160210_1530.py (hunk 3 above)
Comment (kave) [inline cfgov/v1/migrations/0045_auto_20160210_1530.py]: donzo

File: cfgov/v1/migrations/0045_auto_20160210_1530.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+import v1.models.snippets
+import wagtail.wagtailcore.blocks
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('v1', '0044_auto_20160208_1703'),
+    ]
+
+    operations = [
```
Comment (jimmynotjim) [inline cfgov/v1/migrations/0045_auto_20160210_1530.py]: You don't need to break on every `,` maybe just the top level `(),`, just so it's easier to scan through for mistakes.

File: cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html [hunk 5]
```diff
@@ -64,72 +65,22 @@ <h2 class="header-slug">
...
+        {% endif %}
 
-        <div class="o-sidebar-contact-info_column">
-            {% if page.contact.contact_info %}
-                {% for block in page.contact.contact_info %}
-                    {{ render_stream_child(block) }}
-                {% endfor %}
-            {% endif %}
-        </div>
+        {{ value.contact.body | safe }}
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html]: Should this also be wrapped in an if since the body text isn't required?

**Comment by KimberlyMunoz** [comment]
Other than the minor comments above, I tested this in wagtail and it looks great! Thanks!

image

File: cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html (hunk 5 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html]: @KimberlyMunoz nah this `body` field is within the contact model which is required for the organism. but that brings up another point this organism also has its own body field `value.body` thats never used in this template, the same goes for `main-contact-info` is this something we want to render? or remove?

**Pull request merged**